### DISCUSSION TITLE
What Version of Blender do I use this with currently? 3/1/2022

### DISCUSSION THREAD

**Discussion by JeraWolfe** [bnpr/Malt#262] [Q&A]
Really quick, since it just says latest release of Blender, but there are multiple latest releases of blender...
Experimental.
Latest Stable Build.
2.9 is still being updated.

Which one do I use?

**Comment by pragma37** [comment]
The latest version released (3.0 at the moment).
Basically, whatever version is shown here at the top: https://www.blender.org/download/

**Comment by JeraWolfe** [comment]
That is the MOST AWESOMEST ANSWER I COULD GET!!! I’m gonna try it!
Thank you guys, I’ll have something to DONATE soon!

Keep up the amazing work, eh?

All our love and respect from those of us at Digital Dynamics 3d!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
